doc/source/usage.rst

```diff
@@ -3,6 +3,9 @@ Usage
 
 This section illustrates how to run *ksrates* on the use case dataset proposed in the :ref:`explained_example`, where the rate-adjustment is relative to the focal species oil palm (*Elaeis guineensis*). The use case dataset is stored in the GitHub repository under the ``example`` directory. The pipeline steps can be run either through Nextflow (recommended) or manually. In either case it is advised to use a computing cluster. 
 
+.. note::
+    WSL2 users can enter the Windows file system (e.g. ``Documents``) from the terminal through ``cd mnt/c/Users/your_username``.
```

Comment:
Can we assume it's always "c/Users/your_username" and that exists or shall we note it's an example?